apps/transport/lib/transport_web/api/schemas.ex

```diff
@@ -20,13 +20,19 @@ defmodule TransportWeb.API.Schemas do
           type: :string,
           enum: ["Point", "LineString", "Polygon", "MultiPoint", "MultiLineString", "MultiPolygon"]
         }
-      },
-      discriminator: %Discriminator{
-        propertyName: "type"
       }
     }
   end
 
+  defmodule NumberItems do
```

Comment:
by curiosity why did you change this?

Reply:
Because it was compatible with the new version of openapispex